### PR TITLE
Add Night-Owl Theme

### PR DESCRIPTION
**Pull request by yashguptaz**
Add Sarah Drasner's Night Owl Theme to GitHub Readme Stats.

### CODE REVIEW

**Comment by yashguptaz** [comment]
@anuraghazra Is there a way to test the theme really quickly?
Can we have a bot that comments the theme in a markdown format for theme related PRs?

File: themes/index.js [hunk 1]
```diff
@@ -94,7 +94,13 @@ const themes = {
     icon_color: "FB94FF",
     text_color: "A599E9",
     bg_color: "2D2B55",
-  }
+  },
+  'night-owl': {
+    title_color: "#7FDBCA",
+    icon_color: "#FFEB95",
+    text_color: "#C792EA",
+    bg_color: "#011627",
```
Comment (anuraghazra) [inline themes/index.js:102]: @yashguptaz don't need the #

**Comment by anuraghazra** [comment]
> @anuraghazra Is there a way to test the theme really quickly?
> Can we have a bot that comments the theme in a markdown format for theme related PRs?

@yashguptaz 

```md
https://github-readme-stats.vercel.app/api?username=yashguptaz&title_color=7FDBCA&text_color=C792EA&icon_color=FFEB95&bg_color=011627&show_icons=true
```

**Comment by yashguptaz** [comment]
Woah this looks amazing!

File: themes/index.js [hunk 2]
```diff
@@ -94,7 +94,13 @@ const themes = {
     icon_color: "FB94FF",
     text_color: "A599E9",
     bg_color: "2D2B55",
-  }
+  },
+  'night-owl': {
```
Comment (anuraghazra) [inline themes/index.js:98]: I think we can skip the hyphen, "nightowl" is fine i guess

**Comment by yashguptaz** [comment]
@anuraghazra Made all the recommended changes

**Comment by yashguptaz** [comment]
In case you want try out more colors,
here are all the colors of night owl theme.

```
    {
      "name": "Night Owl",
      "background": "#011627",
      "black": "#011627",
      "blue": "#82AAFF",
      "brightBlack": "#575656",
      "brightBlue": "#82AAFF",
      "brightCyan": "#7FDBCA",
      "brightGreen": "#22DA6E",
      "brightPurple": "#C792EA",
      "brightRed": "#EF5350",
      "brightWhite": "#FFFFFF",
      "brightYellow": "#FFEB95",
      "cyan": "#21C7A8",
      "foreground": "#D6DEEB",
      "green": "#22DA6E",
      "purple": "#C792EA",
      "red": "#EF5350",
      "white": "#ffffff",
      "yellow": "#ADDB67"
  }
```

**Comment by anuraghazra** [comment]
> In case you want try out more colors,
> here are all the colors of night owl theme.
> 

Yeah tried them, we don't have lots of tokens so just the primary ones matters anyways. and this looks good.

**Comment by anuraghazra** [comment]
@yashguptaz Thanks for the theme contribution.

**Pull request merged**